test_queue.html

```diff
@@ -0,0 +1,113 @@
+			(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+			m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+			})(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+
+			ga('create', 'UA-65099719-1', 'auto');
+			ga('send', 'pageview');
+
+		</script>
+
+		<script>
+			// Reduce the effect of FOUC (init.js always sets html display to block)
+			var html_el = document.getElementsByTagName('html')[0];
+			html_el.style.display = 'none';
+		</script>
+
+		<noscript>
+			<link rel="stylesheet" href="css/skel.css" />
+			<link rel="stylesheet" href="css/style.css" />
+			<link rel="stylesheet" href="css/style-desktop.css" />
+			<link rel="stylesheet" href="css/style-noscript.css" />
+		</noscript>
+		<!--[if lte IE 8]><link rel="stylesheet" href="css/ie/v8.css" /><![endif]-->
+	</head>
+	<body class="no-sidebar">
+
+		<!-- Header -->
+			<div id="header">
+
+				<!-- Inner -->
+					<div class="inner">
+						<header>
+							<h1><a href="index.html" id="logo">Integrated Microbiome Resource (IMR)</a></h1>
+						</header>
+					</div>
+
+				<!-- Nav -->
+					<nav id="nav">
+						<ul>
+							<li><a href="index.html">Home</a></li>
+							<li><a href="about.html">About Us</a></li>
+							<li><a href="achieve.html">Achievements</a></li>
+							<li><a href="submissions.html">Submission Guidelines</a></li>
+							<li><a href="queue.html">Real-Time Queue</a></li>
+							<li><a href="pricing.html">Pricing</a></li>
+							<li><a href="protocols.html">Protocols</a></li>
+							<li><a href="contact.html">Contact Us</a></li>
+						</ul>
+					</nav>
+
+			</div>
+
+		<!-- Main -->
+			<div class="wrapper style1">
+
+				<div class="container">
+					<article id="main" class="special">
+						<header>
+							<h2>Real-Time Sample Queue</h2>
+						</header>
+						<section>
+							<p>We will attempt to maintain as close to real-time as possible project/sample information in this queue display below. This will allow you to monitor the progress of your samples
+							through the IMR workflow. It will also allow prospective clients to get a sense of how full the current queued sequencing runs are for those who wish to multiplex without
+							contributing whole runs (as discussed on our submissions page).
+							<h3><font style="color:red">Warning: Due to high demand, sample queue is quite full, extending typical data delivery time to ~3 months.</font></h3>
+							<p></p>
+
+							<div id="queue-app-root"></div>
+
+						</section>
+					</article>
+					</div>
+				</div>
+
+		<!-- Footer -->
+			<div id="footer">
+							<!-- Copyright -->
+								<div class="copyright">
+									<ul class="menu">
+										<li>&copy; CGEB-IMR (2014+). All rights reserved.</li><li>Design template: <a href="http://html5up.net" target="new">HTML5 UP</a></li>
+									</ul>
+								</div>
+
+			</div>
+
+		<script src="js/main.605511c8.dev.js"></script>
```

Comment:
For the production version, the JS file won't have the `dev` in it. The JS file(s) will hosted by GitHub for now, sticking with established convention.